Sources/Automerge/Document.swift

```diff
@@ -58,13 +58,13 @@ public final class Document: @unchecked Sendable {
     /// The actor ID of this document
     public var actor: ActorId {
         get {
-            lock {
-                ActorId(ffi: self.doc.wrapErrors { $0.actorId() })
+            read {
+                ActorId(ffi: doc.actorId() )
             }
         }
         set {
-            lock {
-                self.doc.wrapErrors { $0.setActor(actor: [UInt8](newValue.data)) }
+            write {
```

Comment:
Should setting an Acor trigger change events?

Reply:
If it's changing the underlying document structure and content in any way, then yes.